### PR TITLE
fix(alert-provider):fix alert provider ,add severities

### PR DESCRIPTION
**Pull request by benben12341**

### CODE REVIEW

File: client/App/Providers/AlertProvider/index.js [hunk 1]
```diff
@@ -3,17 +3,26 @@ import Alert from './Snackbar';
...
+  warning: 'warning',
+};
+
+export default (props) => {
   const [message, setMessage] = useState(null);
   const [opened, setOpened] = useState(false);
+  const [severity, setSeverity] = useState(severities.success);
 
-  const open = ({ message }) => {
+  const open = (message, severity) => {
```
Comment (shirmagen) [inline client/App/Providers/AlertProvider/index.js]: Why you deleted the destruction?

File: client/App/Providers/AlertProvider/index.js [hunk 2]
```diff
@@ -23,11 +32,12 @@ export default props => {
...
       value={{
         close,
-        open
+        open,
+        severities,
       }}
       {...props}
     >
-      <Alert message={message} opened={opened} handleClose={close} />
+      <Alert message={message} opened={opened} handleClose={close} severity={`${severity}`} />
```
Comment (shirmagen) [inline client/App/Providers/AlertProvider/index.js]: Why in text? Just do severity={severity}

**Review by shirmagen — CHANGES_REQUESTED** [review]

File: client/App/Providers/AlertProvider/index.js [hunk 3]
```diff
@@ -3,17 +3,26 @@ import Alert from './Snackbar';
...
+  warning: 'warning',
+};
+
+export default (props) => {
   const [message, setMessage] = useState(null);
   const [opened, setOpened] = useState(false);
+  const [severity, setSeverity] = useState(severities.success);
 
-  const open = ({ message }) => {
+  const open = ({ message, severity }) => {
```
Comment (OmerDital) [inline client/App/Providers/AlertProvider/index.js:17]: onOpen

File: client/App/Providers/AlertProvider/index.js [hunk 4]
```diff
@@ -3,17 +3,26 @@ import Alert from './Snackbar';
...
     setMessage(message);
     setOpened(true);
+    setSeverity(severity);
   };
 
   const close = () => {
```
Comment (OmerDital) [inline client/App/Providers/AlertProvider/index.js:23]: onClose

**Review by OmerDital — APPROVED** [review]

**Pull request merged**